pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -79,8 +80,8 @@
             pnpm install \
                 --force \
                 --ignore-scripts \
-                ${installFlags} \
```

Comment:
Hmm OK I guess it'd simply have been more correct to remove that in the previous commit.